script.js

```diff
@@ -1,4 +1,135 @@
+  if (!userName) {
+    if (!input) {
+      return "Please enter your username";
+    }
+    userName = input;
+    return `Thanks ${userName} for entering your name. To start playing, please type "scissors" "paper" or "stone".`;
+  }
+  if (input == PAPER && computerChoiceMade == PAPER) {
+    totalCount = totalCount + 1;
+    drawCount = drawCount + 1;
+    return (
+      "The computer chose paper.<br>You chose paper.<br>You draw. You have won " +
+      winCount +
+      "/" +
+      totalCount +
+      " games."
+    );
+  }
+  if (input == PAPER && computerChoiceMade == STONE) {
+    winCount = winCount + 1;
+    totalCount = totalCount + 1;
+    return (
+      "The computer chose stone.<br>You chose paper.<br>You win. You have won " +
+      winCount +
+      "/" +
+      totalCount +
+      " games."
+    );
+  }
+  if (input == PAPER && computerChoiceMade == SCISSORS) {
+    loseCount = loseCount + 1;
+    totalCount = totalCount + 1;
+    return (
+      "The computer chose scissors.<br>You chose paper.<br>You lose. You have won " +
+      winCount +
+      "/" +
+      totalCount +
+      " games."
+    );
+  }
+  if (input == STONE && computerChoiceMade == PAPER) {
+    loseCount = loseCount + 1;
+    totalCount = totalCount + 1;
+    return (
+      "The computer chose paper.<br>You chose stone.<br>You lose. You have won " +
+      winCount +
+      "/" +
+      totalCount +
+      " games."
+    );
+  }
+  if (input == STONE && computerChoiceMade == SCISSORS) {
+    winCount = winCount + 1;
+    totalCount = totalCount + 1;
+    return (
+      "The computer chose scissors.<br>You chose stone.<br>You win. You have won " +
+      winCount +
+      "/" +
+      totalCount +
+      " games."
+    );
+  }
+  if (input == STONE && computerChoiceMade == STONE) {
+    totalCount = totalCount + 1;
+    drawCount = drawCount + 1;
+    return (
+      "The computer chose stone.<br>You chose stone.<br>You draw. You have won " +
+      winCount +
+      "/" +
+      totalCount +
+      " games."
+    );
+  }
+  if (input == SCISSORS && computerChoiceMade == PAPER) {
+    winCount = winCount + 1;
+    totalCount = totalCount + 1;
+    return (
+      "The computer chose paper.<br>You chose scissors.<br>You win. You have won " +
+      winCount +
+      "/" +
+      totalCount +
+      " games."
+    );
+  }
+  if (input == SCISSORS && computerChoiceMade == STONE) {
+    loseCount = loseCount + 1;
+    totalCount = totalCount + 1;
+    return (
+      "The computer chose stone.<br>You chose scissors.<br>You lose. You have won " +
+      winCount +
+      "/" +
+      totalCount +
+      " games."
+    );
+  }
+  if (input == SCISSORS && computerChoiceMade == SCISSORS) {
+    totalCount = totalCount + 1;
+    drawCount = drawCount + 1;
+    return (
+      "The computer chose scissors.<br>You chose scissors.<br>You draw. You have won " +
+      winCount +
+      "/" +
+      totalCount +
+      " games."
+    );
+  }
```

Comment:
For now each win/loss condition is enumerated and the output values produced one by one, why is quite a lot of code duplication
Can consider grouping all the win condtions under one if condition using ||
e.g. 
if (
(input == SCISSORS && computerChoiceMade == SCISSORS) ||
(input == SCISSORS && computerChoiceMade == STONE)  ||
(input == SCISSORS && computerChoiceMade == PAPER))

and puting the inputs and computerChoiceMade as template literals in your returned string

`The computer chose ${computerChoiceMade}........You chose ${input}`